Gemfile

```diff
@@ -2,3 +2,17 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in task.gemspec
 gemspec
+
+gem 'cassava', :git => 'git@github.com:backupify/cassava.git'
+gem 'pyper', :git => 'git@github.com:backupify/pyper.git'
+
+group :development, :test do
+  gem "pry"
+  gem "awesome_print"
+  gem 'm', :git => 'git@github.com:ANorwell/m.git', :branch => 'minitest_5'
```

Comment:
m is a gem for running minitest tests, allows running a single test by line number and stuff like that: https://github.com/qrush/m

I updated it to work with minitest 5.